test/e2e/pods.go

```diff
@@ -1075,6 +1075,34 @@ var _ = framework.KubeDescribe("Pods", func() {
 		}, 0, defaultObservationTimeout)
 	})
 
+	It("should be restarted with a docker exec liveness probe with timeout [Conformance]", func() {
+		runLivenessTest(f.Client, f.Namespace.Name, &api.Pod{
+			ObjectMeta: api.ObjectMeta{
+				Name:   "liveness-exec",
+				Labels: map[string]string{"test": "liveness"},
+			},
+			Spec: api.PodSpec{
+				Containers: []api.Container{
+					{
+						Name:    "liveness",
+						Image:   "gcr.io/google_containers/busybox:1.24",
+						Command: []string{"/bin/sh", "-c", "sleep 600"},
+						LivenessProbe: &api.Probe{
+							Handler: api.Handler{
+								Exec: &api.ExecAction{
+									Command: []string{"/bin/sh", "-c", "sleep 10"},
+								},
+							},
+							InitialDelaySeconds: 15,
+							TimeoutSeconds:      1,
+							FailureThreshold:    1,
+						},
+					},
+				},
+			},
+		}, 1, defaultObservationTimeout)
```

Comment:
Looked at the helper, it's indeed **at least n** semantics.
[test/e2e/pods.go#L104-L110](https://github.com/kubernetes/kubernetes/blob/5e7c309b29280bef441800953a33a6468a0dc0c4/test/e2e/pods.go#L104-L110)

I think checking for at least 1 restart (in the period of 2 minutes, as per `defaultObservationTimeout`) should be enough. @ncdc @vishh do you agree?